conda/gateways/repodata/__init__.py

```diff
@@ -57,14 +55,21 @@ class Response304ContentUnchanged(Exception):
 
 
 class CondaRepoInterface(RepoInterface):
-    def __init__(self, url: str, repodata_fn: str | None, **kwargs) -> None:
```

Comment:
It takes **kwargs because we want to pass extra properties to the subclass in features/repodata-jlap